cartography/intel/aws/ec2/load_balancer_v2s.py

```diff
@@ -159,6 +159,7 @@ def load_load_balancer_v2_target_groups(
     MERGE (elbv2)-[r:EXPOSE]->(instance)
     ON CREATE SET r.firstseen = timestamp()
     SET r.lastupdated = $update_tag
+        r.port = $PORT, r.protocol = $PROTOCOL
```

Comment:
You missed a comma. Needs to be

```
    SET r.lastupdated = $update_tag,
        r.port = $PORT, r.protocol = $PROTOCOL
```

Also, is the ARN of the target group available? If so, maybe we should include the ARN of the target group as `r.target_group_arn` so that it's easily searchable.

Reply:
Oh also let's remember to update the schema doc